src/actions/actions.test.js

```diff
@@ -0,0 +1,168 @@
+import * as actions from './index';
+
+describe('actions', () => {
+  it('should return a type of addAllProjects with a projects array', () => {
+    const projects = [
+      {
+        name:'Test',
+        id: 1
+      }
+    ]
+    const expected = {
+      type: 'ADD_ALL_PROJECTS',
+      projects
+    }
+
+    const result = actions.addAllProjects(projects)
+
+    expect(result).toEqual(expected)
+  });
+  it('should return a type of addAllPalettes with a palettes array', () => {
+    const palettes = [
+      {
+        id:1,
+        project_id:3,
+        name: 'test palette',
+        color_1: 'A8E0FF',
+        color_2: 'A8E0FF',
+        color_3: 'A8E0FF',
+        color_4: 'A8E0FF',
+        color_5: 'A8E0FF',
+      }
+
+    ]
+    const expected = {
+      type: 'ADD_ALL_PALETTES',
+      palettes
+    }
+
+    const result = actions.addAllPalettes(palettes)
+
+    expect(result).toEqual(expected)
+  })
+  it('should return a type of setLoading with a boolean', () => {
+    const bool = true;
+    const expected = {
+      type: 'SET_LOADING',
+      isLoading: bool
+    }
+
+    const result = actions.setLoading(bool);
+
+    expect(result).toEqual(expected);
+  });
+  it('should return a type of addProject with a new project', () => {
+    const project = {
+        name: 'Test',
+        id: 1
+      }
+
+    
+    const expected = {
+      type: 'ADD_PROJECT',
+      project
+    }
+
+    const result = actions.addProject(project)
+
+    expect(result).toEqual(expected)
+  });
+  it('should return a type of addPalette with a new palette', () => {
+    const palette = {
+        id: 1,
+        project_id: 3,
```

Comment:
Maybe we should set this as a global variable or beforeEach if it's going to be used multiple times